sessionctx/variable/noop.go

```diff
@@ -128,7 +128,7 @@ var noopSysVars = []*SysVar{
 	{Scope: ScopeNone, Name: "skip_networking", Value: "0"},
 	{Scope: ScopeGlobal, Name: "innodb_monitor_reset", Value: ""},
 	{Scope: ScopeNone, Name: "ssl_cipher", Value: ""},
-	{Scope: ScopeNone, Name: "tls_version", Value: "TLSv1,TLSv1.1,TLSv1.2"},
+	{Scope: ScopeNone, Name: "tls_version", Value: getAvailableTLSVersion()},
```

Comment:
Please move it from `noop.go` to `sysvar.go` if it now does something correct :-)